Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+
+function CardBanner() {
+  return (
+    <img src = "https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png">  </img>
```

Comment:
Image tags are self-closing. Just write it like this:

```suggestion
    <img src = "https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" />
```